microprofile/4.0/config/2.0/TCKResults.adoc

```diff
@@ -7,7 +7,7 @@ As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Tech
 
 * Product Name, Version and download URL (if applicable):
 +
-https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/release/2021-03-09_1101/openliberty-all-21.0.0.3-cl210320210309-1101.zip[Open Liberty 21.0.0.3]
+https://mvnrepository.com/artifact/io.openliberty/openliberty-runtime/21.0.0.3[Open Liberty 21.0.0.3]
```

Comment:
mvnrepository.com isn't the right place to link to (it's just an index on top of maven central). Should be either dhe.ibm.com or maven.org.

either
https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/release/2021-03-09_1101/openliberty-21.0.0.3.zip
or maybe
https://repo1.maven.org/maven2/io/openliberty/openliberty-runtime/21.0.0.3/openliberty-runtime-21.0.0.3.zip

Reply:
ah, ok. I will use the maven repo.

Reply:
done now.